source/administration/config-settings.rst

```diff
@@ -2320,9 +2320,18 @@ This setting defines the frequency of cluster request time logging for :doc:`../
 | This feature's ``config.json`` setting is ``"ClusterLogTimeoutMilliseconds": 2000`` with whole number input.                                                         |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Enable Searching of Posts
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If this setting is enabled, users can search messages. If it's disabled, users get an error message when they attempt to use the search box.
```

Comment:
1) Maybe add a note it's for improving performance?

E.g. if lots of users do searching in a deployment with millions of messages, it has the potential of causing the server to slow down

2) Also propose adding a note that if you're looking to improve performance, you can first try to use the search read replicas, then disable search